micro_dl/config_preprocess_kidney_confocal.yml

```diff
@@ -4,16 +4,18 @@ input_dir: '/data/sguo/Processed/2018_11_01_kidney_slice/SMS_2018_1101_1906_1_BG
 num_workers: 4
 correct_flat_field: False
 uniform_structure: True
+create_masks: False
 masks:
     channels: [5]
     str_elem_radius: 3
+do_tiling: True
 tile:
     channels: -1
     tile_size: [256, 256]
     step_size: [128, 128]
     depths: [3, 3, 3, 3, 1, 1, 1]
     mask_depth: 1
-    image_format: 'zyx'
+    shape_order: 'zyx'
```

Comment:
i think we changed this to image_format!

Reply:
I'll delete this old config file as we don't need it in the repo